### ISSUE TITLE
Is there a license

### ISSUE THREAD

**Issue by ugintl** [oceceli/last-automation#1]
Does it have a license?

**Comment by oceceli** [comment]
It has an MIT license. use it however you want if you like to 💯

**Comment by ugintl** [comment]
@oceceli Thank you for your reply. Does your app have these functionalities?

To be able allow for multiple work orders and multiple users to be open simultaneously. The system will also need to calculate productivity for individual manufacturers based on standard manufacturing time as well as generate new average production times based on data inputted. Having a clock in clock out function on work orders would also be helpful.

**Comment by oceceli** [comment]
Hi @ugintl, thank you for your interest.

This project was my very first project in order to be a reference project for me to hop into a job in the industry. These functionalities that you asked for are not implemented and will not be implemented anytime soon(these were on the to-do list though). The tech stack that I used in this project is Laravel and Livewire. After a while, I realized Livewire is absolutely not a good fit for this kind of project. It should be an API or Inertia with a JS framework. It was too late, so I decided to end it as it already took so much time for a non-profit project. 

I don't think you can use it in a production environment because it needs a lot of work to do. But rather, you can get inspired by it, use some entities of it, and even create pull requests on it! 

Don't hesitate to ask me about the project whenever you need.

**Comment by ugintl** [comment]
@oceceli just a general question. I am always confused with licenses. Sometimes they are tricky. If an app is MIT, does it mean that I can use my own branding and sell it. Like propriety software?

**Comment by oceceli** [comment]
@ugintl yes! The beauty of the MIT license is that it's 'free'. I mean free, "free". You can make a million dollars of it. Nobody will interrupt you. Feel free to do anything without hesitation. 
But it's a very common thing among software developers that if you can make money with it you should buy a cup of coffee for its owner as gratitude :)

**Comment by ugintl** [comment]
@oceceli understood and agreed. Developers put a lot of effort.
